Error Messages in building doc for kinetic.

```
writing... 
Exception occurred:
  File "/usr/local/lib/python2.7/dist-packages/docutils/nodes.py", line 625, in __getitem__
    return self.attributes[key]
KeyError: 'prefix'
The full traceback has been saved in /tmp/sphinx-err-AN4mhX.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
CMakeFiles/docbuild_tork_moveit_tutorial_kinetic.dir/build.make:57: recipe for target 'CMakeFiles/docbuild_tork_moveit_tutorial_kinetic' failed
make[3]: *** [CMakeFiles/docbuild_tork_moveit_tutorial_kinetic] Error 2
CMakeFiles/Makefile2:323: recipe for target 'CMakeFiles/docbuild_tork_moveit_tutorial_kinetic.dir/all' failed
make[2]: *** [CMakeFiles/docbuild_tork_moveit_tutorial_kinetic.dir/all] Error 2
CMakeFiles/Makefile2:330: recipe for target 'CMakeFiles/docbuild_tork_moveit_tutorial_kinetic.dir/rule' failed
make[1]: *** [CMakeFiles/docbuild_tork_moveit_tutorial_kinetic.dir/rule] Error 2
Makefile:277: recipe for target 'docbuild_tork_moveit_tutorial_kinetic' failed
make: *** [docbuild_tork_moveit_tutorial_kinetic] Error 2

Exited with code exit status 2
```

--- a/doc/moveit-tutorial_ja_reference-class-functions.md
+++ b/doc/moveit-tutorial_ja_reference-class-functions.md
@@ -5,10 +5,9 @@
 
 ### MoveGroupCommander クラス
 
-- GitHub - ros_planning/moveit_commander/src/moveit_commander/move_group.py
-  - [https://github.com/ros-planning/moveit_commander/blob/<$ROS_DISTRO>-devel/src/moveit_commander/move_group.py][67977aee]
-- GitHub -  ros_planning/moveit/moveit_ros/planning_interface/move_group_interface/src/wrap_python_move_group.cpp
-  -  [https://github.com/ros-planning/moveit/blob/<$ROS_DISTRO>-devel/moveit_ros/planning_interface/move_group_interface/src/wrap_python_move_group.cpp][7a957dfd]
+GitHub - [moveit_commander/src/moveit_commander/move_group.py][67977aee]
+
+GitHub - [moveit/moveit_ros/planning_interface/move_group_interface/src/wrap_python_move_group.cpp][7a957dfd]
 
   [67977aee]: https://github.com/ros-planning/moveit_commander/blob/<$ROS_DISTRO>-devel/src/moveit_commander/move_group.py "move_group.py"
   [7a957dfd]: https://github.com/ros-planning/moveit/blob/<$ROS_DISTRO>-devel/moveit_ros/planning_interface/move_group_interface/src/wrap_python_move_group.cpp "wrap_python_move_group.cpp"
@@ -707,11 +706,9 @@
 
 ### RobotCommander クラス
 
-- GitHub -  ros-planning/moveit_commander/src/moveit_commander/robot.py
-  - [https://github.com/ros-planning/moveit_commander/blob/<$ROS_DISTRO>-devel/src/moveit_commander/robot.py][ad535b3a]
-- GitHub - ros-planning/moveit/moveit_ros/planning_interface/robot_interface/src/wrap_python_robot_interface.cpp
+GitHub - [moveit_commander/src/moveit_commander/robot.py][ad535b3a]
 
-  - [https://github.com/ros-planning/moveit/blob/<$ROS_DISTRO>-devel/moveit_ros/planning_interface/robot_interface/src/wrap_python_robot_interface.cpp][4cfc3b2d]
+GitHub - [moveit/moveit_ros/planning_interface/robot_interface/src/wrap_python_robot_interface.cpp][4cfc3b2d]
 
   [ad535b3a]: https://github.com/ros-planning/moveit_commander/blob/<$ROS_DISTRO>-devel/src/moveit_commander/robot.py "RobotCommander Class"
 
@@ -972,11 +969,10 @@
 
 ### PlanningSceneInterface クラス
 
-- GitHub - ros-planning/moveit_commander/src/moveit_commander/planning_scene_interface.py
-  - [https://github.com/ros-planning/moveit_commander/blob/<$ROS_DISTRO>-devel/src/moveit_commander/planning_scene_interface.py][844eea07]
-- GitHub -  ros-planning/moveit/moveit_ros/planning_interface/planning_scene_interface/src/wrap_python_planning_scene_interface.cpp
+GitHub - [moveit_commander/src/moveit_commander/planning_scene_interface.py][844eea07]
 
-  - [https://github.com/ros-planning/moveit/blob/<$ROS_DISTRO>-devel/moveit_ros/planning_interface/planning_scene_interface/src/wrap_python_planning_scene_interface.cpp][86935f4f]
+GitHub - [moveit/moveit_ros/planning_interface/planning_scene_interface/src/wrap_python_planning_scene_interface.cpp][86935f4f]
+
   [844eea07]: https://github.com/ros-planning/moveit_commander/blob/<$ROS_DISTRO>-devel/src/moveit_commander/planning_scene_interface.py "GitHub - PlanningSceneInterface クラス"
 
   [86935f4f]: https://github.com/ros-planning/moveit/blob/<$ROS_DISTRO>-devel/moveit_ros/planning_interface/planning_scene_interface/src/wrap_python_planning_scene_interface.cpp "wrap_python_planning_scene_interface.cpp"
@@ -1194,8 +1190,7 @@
 
 ### conversions の関数
 
-- GitHub - ros-planning/moveit_commander/src/moveit_commander/conversions.py
-  - [https://github.com/ros-planning/moveit_commander/blob/<$ROS_DISTRO>-devel/src/moveit_commander/conversions.py][0aa62cbe]
+- GitHub - [moveit_commander/src/moveit_commander/conversions.py][0aa62cbe]
 
   [0aa62cbe]: https://github.com/ros-planning/moveit_commander/blob/<$ROS_DISTRO>-devel/src/moveit_commander/conversions.py "conversions functions"
 
@@ -1277,8 +1272,7 @@
 
 ### roscpp_initializer の関数
 
-- GitHub - ros-planning/moveit_commander/src/moveit_commander/roscpp_initializer.py
-  - [https://github.com/ros-planning/moveit_commander/blob/<$ROS_DISTRO>-devel/src/moveit_commander/roscpp_initializer.py][f6bf1bd0]
+- GitHub - [moveit_commander/src/moveit_commander/roscpp_initializer.py][f6bf1bd0]
 
   [f6bf1bd0]: https://github.com/ros-planning/moveit_commander/blob/<$ROS_DISTRO>-devel/src/moveit_commander/roscpp_initializer.py "roscpp_initializer functions"
 

--- a/doc/moveit-tutorial_ja_robot-python_basic.md
+++ b/doc/moveit-tutorial_ja_robot-python_basic.md
@@ -203,9 +203,10 @@ Out[32]: True
 どれを渡しても `set_pose_target()` 内で判断して適切な処理がなされます．
 
 - 参考 : GitHub - move_group.py 内の `set_pose_target()` の定義
-  - [https://github.com/ros-planning/moveit/blob/35a94c96ab21e8c8c2994adbf47cc7e39e30cfe7/moveit_commander/src/moveit_commander/move_group.py#L252][6cf69e4b]
 
-  [6cf69e4b]: https://github.com/ros-planning/moveit/blob/35a94c96ab21e8c8c2994adbf47cc7e39e30cfe7/moveit_commander/src/moveit_commander/move_group.py#L252 "set_pose_target()"
+[https://github.com/ros-planning/moveit/blob/1.0.2/moveit_commander/src/moveit_commander/move_group.py#L252][6cf69e4b]
+
+  [6cf69e4b]: https://github.com/ros-planning/moveit/blob/1.0.2/moveit_commander/src/moveit_commander/move_group.py#L252 "set_pose_target()"
 
 それでは右手先の位置と姿勢を指定して腕を動かしてみます．
 まずは 位置とRPY角 を `set_pose_target()` に渡して動作させます．

--- a/doc/moveit-tutorial_ja_running-your-own-program.md
+++ b/doc/moveit-tutorial_ja_running-your-own-program.md
@@ -306,13 +306,13 @@ TORK MoveIt! Tutorial パッケージを最初にインストールしている�
 $ rospack find tork_movieit_tutorial
 ```
 
-1. ワークスペース内の `tork_moveit_tutorial` が **参照されている** 場合
+- ワークスペース内の `tork_moveit_tutorial` が **参照されている** 場合
     （パッケージの取得とビルドを行った方を参照）
     ```
     robotuser@robotuser-PC:~/catkin_ws$ rospack find tork_moveit_tutorial 
     /home/robotuser/catkin_ws/src/tork_moveit_tutorial
     ```
-1. ワークスペース内の `tork_moveit_tutorial` が **参照されていない** 場合
+- ワークスペース内の `tork_moveit_tutorial` が **参照されていない** 場合
     （元々インストールされているパッケージの方を参照）
     ```
     robotuser@robotuser-PC:~/catkin_ws$ rospack find tork_moveit_tutorial 

